pages/verify-identity.tsx

```diff
@@ -135,7 +135,12 @@ export default function IdentityVerificationForm() {
             </FormControl>
             <FormControl isRequired textAlign="left" marginBottom="20px">
               <FormLabel>{`Date of Birth`}</FormLabel>
-              <DatePicker width="184px" value={dateOfBirth} onDateChange={setDateOfBirth} />
+              <Input
+                type="date"
+                width="184px"
+                value={dateOfBirth}
+                onChange={event => setDateOfBirth(event.target.value)}
+              />
               <FormHelperText>
                 {`Please enter your date of birth in YYYY-MM-DD format. For example, if you were born on
             20th August 1950, you would enter 20-08-1950`}
```

Comment:
```suggestion
            20th August 1950, you would enter 1950-08-20`}
```

Reply:
Will leave this for now as it's subject to change!